src/GregClient/Utility/Utilities.cs

```diff
@@ -19,11 +37,7 @@ static DebugLogger()
         {
             try
             {
-                var enableDebugLogsKey = "EnableDebugLogs";
-
-                var dllPath = new Uri(Assembly.GetExecutingAssembly().GetName().CodeBase).LocalPath;
-                var config = ConfigurationManager.OpenExeConfiguration(dllPath);
-                var enableDebugLogsSetting = config.AppSettings.Settings[enableDebugLogsKey];
+                var enableDebugLogsSetting = AppSettingMgr.GetItem("EnableDebugLogs");
```

Comment:
can we add a sample of this config file?

Reply:
Do we want to include the config file in the nuget package ?
I was trying to avoid adding the Timeout to the public package...so that users are not tempted to add crazy values...